### PR TITLE
Document changed_when clause

### PR DESCRIPTION
**Pull request by stoned**
Documentation update related to #3524

### CODE REVIEW

File: docsite/latest/rst/playbooks2.rst [hunk 1]
```diff
@@ -69,6 +69,31 @@ write a task that looks like this::
...
+Overriding Changed Result
+`````````````````````````
+
+.. versionadded:: 1.3
+
+When a task make some changes or sometimes is simply executed, it
+is reported as changed.  You may want to override this.  To do so,
+write a task with the `changed_when` clause, which actually is a
+Python expression. When this expression evaluates to true the task
+is considered changed, when it evaluates to false the tasks is
```
Comment (pol) [inline docsite/latest/rst/playbooks2.rst:81]: This is confusing or wrong, is it considered changed when it is true or false?

File: docsite/latest/rst/playbooks2.rst [hunk 2]
```diff
@@ -69,6 +69,31 @@ write a task that looks like this::
       action: command /bin/false
       ignore_errors: yes
 
+Overriding Changed Result
+`````````````````````````
+
+.. versionadded:: 1.3
+
+When a task make some changes or sometimes is simply executed, it
```
Comment (pol) [inline docsite/latest/rst/playbooks2.rst:77]: Grammar is clumsy, recommended change:

Use the "changed_when" option to override the default module "change" reporting.  The "changed_when" option takes a Python expression.  ...

**Comment by mpdehaan** [comment]
@pol -- This is "confusing or wrong" is a bit harsh.

I'm going to merge this and make an edit pass.

**Comment by mpdehaan** [comment]
merged, thank you!

**Comment by pol** [comment]
@mdehaan, fair enough, sorry for the harshness. I sometimes make the mistake of sacrificing politeness for brevity. Apologies! It is a great addition.

**Comment by stoned** [comment]
No worries, my grammar could use a helping hand. Thanks !

**Pull request merged**